new_platforms/vscode-extension/README.md

```diff
@@ -0,0 +1,147 @@
+
+Once your deployment json has been created in the `config` folder, you can deploy
+to an Azure Edge device by:
+
+1. Navigate to the `config` folder and right click on `deployment.*.json`
+1. Select `Create Deployment for Single Device` or `Create Deployment at Scale`.
+
+### Debug your Open Enclave solution.
+
+Debugging your standalone project's enclave is easy.  
+
+1. Set breakpoints in the files you wish to debug.  Breakpoints in the enclave may only be added before
+the emulator (QEMU) starts or when the debugger is already broken inside the enclave.
+1. Choose the architecture you are interested in debugging by navigating to the Visual 
+Studio **Debug** view and selecting either `(gdb) Launch QEMU (ARMv7-A)` or 
+`(gdb) Launch QEMU (AARCH64/ARMv8-A)` from the debug configuration dropdown.
+1. You can simply hit `F5`.  This will configure you cmake, run the build, start QEMU, and load 
+the host and enclave symbols into an instance of the debugger.
+1. Open the **Terminal** view
+1. Log into QEMU using `root` (no password is required)
+1. Start the host process by entering `/mnt/host/bin/<solution-name>`
+
+        Note: The debugger has been configured to break at TA_InvokeCommandEntryPoint.  This will happen once when the enclave starts and once for each ECALL.
+
+## Requirements
+
+* Install [Visual Studio Code](https://code.visualstudio.com/)
+* Install required compilers
+
+      sudo apt update && sudo apt install -y build-essential cmake gcc-arm-linux-gnueabihf gcc-aarch64-linux-gnu g++-arm-linux-gnueabihf g++-aarch64-linux-gnu gdb-multiarch
+
+* Make sure that the [Native Debug extension](https://marketplace.visualstudio.com/items?itemName=webfreak.debug) is installed.
+* Ensure that the requirements are met for the [Azure IoT Edge extension](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.azure-iot-edge):
+    * Ensure that [Docker is installed and running: https://docs.docker.com/get-started/](https://docs.docker.com/get-started/).
+    * Ensure that the [iotedgehubdev](https://pypi.org/project/iotedgehubdev/) tool is installed
+
+            pip install --upgrade iotedgehubdev
+
+    * Create a container repository, like [Azure Container Registry](https://azure.microsoft.com/en-us/services/container-registry/)
+    * Make sure that the [Azure Account extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account) is installed and utilized:
+        1. **CTRL-Shift-P**
+        1. `Azure: Sign In`
+    * Make sure that the [Azure IoT Hub Toolkit](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.azure-iot-toolkit) extension is installed and utilized:
+        1. **CTRL-Shift-P**
+        1. `Azure IoT Hub: Select IoT Hub`
+
+### For development of this extension, or running from source code directly
+
+* Install [node](https://nodejs.org/en/)
+* Install [npm](https://www.npmjs.com/get-npm)
+
+## Data/Telemetry
```

Comment:
The paragraph below uses single spaces after periods whereas the rest of the doc uses double spaces.